### DISCUSSION TITLE
What does the   <EnablePhotos></EnablePhotos> xml tag do in TV and Movie libraries?

### DISCUSSION THREAD

**Discussion by edrock200** [jellyfin/jellyfin#9815] [Q&A] [answered]
Was browsing through the library xmls in root because I wanted to create a script to search for and replace realtime monitoring from true to false and noticed the Movie and TV Show options.xml files start with: <EnablePhotos>true</EnablePhotos>

In browsing the GUI options for these libraries, I don't see any option referring to photos. So I was just curious what this setting does.

**Comment by crobibero** [comment, marked as answer]
This setting is only applied for the "Home Videos" library type (named "Photos" in 10.8 and "Home Videos and Photos" in 10.9)
